website/templates/profile/notifications.mako

```diff
@@ -25,10 +25,19 @@
                         <div class="form-group">
 
                             <input type="checkbox"
-                                   data-bind="checked: subscribed"/>
-                            <label data-bind="text: list"></label>
+                                   data-bind="checked: subscribed" value="Open Science Framework General"/>
+                            <label data-bind="text: list[0]"></label>
```

Comment:
More work than simply updating the contextVars to be a dict. I can take a deeper look into it if you feel it's something worth digging in. I personally don't like using indexes, but they're are hard coded to match the right data, as mailchimp is explicitly the first item on the list (that code is a few lines below where contextVars get extended).